lsst-epo/investigation-static
# [C] update to styling and sectionbreaks

For JIRA: "EPO-7155 #IN-REVIEW #comment update to styling and sectionbreaks"

JIRA: https://jira.lsstcorp.org/browse/EPO-7155

## What this change does ##

- Use finalized icon
- Update second progress bar color
- Add a variation on the section break GIF for the final section break in an investigation

## Notes for reviewers ##

Look for changes on the frontend deployment

Include an indication of how detailed a review you want on a 1-10 scale.
- 1 = "I barely need review on this"

## Testing ##

- Verifying favicon is difficult since favicons are heavily cached
- Verify that the progress bar displays a turquoise as its second color instead of red
- Verify that the final section break has a different image that is available in English and Spanish and other section breaks are unchanged

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
